libraries/spark-job/src/test/scala/uk/gov/ukho/ais/SessionTest.scala

```diff
@@ -0,0 +1,17 @@
+package uk.gov.ukho.ais
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class SessionTest {
+
+  @Test
+  def whenInitAsTestThenGetSessionThenSessionIsCreatedAndNameIsCorrectAndMasterIsLocal()
```

Comment:
Optional: The test whenInitAsTestThenGetSessionThenSessionIsCreatedAndNameIsCorrectAndMasterIsLocal could have a more readable name maybe something like whenInitialisedAsTestAndGetSessionCalledThenSessionIsCreatedAndHasCorrectNameAndMasterIsLocal
```suggestion
  def whenInitialisedAsTestAndGetSessionCalledThenSessionIsCreatedAndHasCorrectNameAndMasterIsLocal()
```